core/src/main/resources/error/error-classes.json

```diff
@@ -134,9 +154,41 @@
     "message" : [ "The second argument of '%s' function needs to be an integer." ],
     "sqlState" : "22023"
   },
+  "TABLE_STATS_MUST_SPECIFIED" : {
+    "message" : [ "table stats must be specified." ],
+    "sqlState" : "42000"
+  },
+  "UNABLE_CREATE_DATABASE_AS_FAILED_CREATE_DIRECTORY" : {
+    "message" : [ "Unable to create database %s as failed to create its directory %s" ]
+  },
+  "UNABLE_CREATE_PARTITION" : {
+    "message" : [ "Unable to create partition path %s" ]
+  },
+  "UNABLE_CREATE_TABLE_AS_FAILED_CREATE_DIRECTORY" : {
+    "message" : [ "Unable to create table %s as failed to create its directory %s" ]
+  },
+  "UNABLE_DELETE_PARTITION" : {
+    "message" : [ "Unable to delete partition path %s" ]
+  },
+  "UNABLE_DROP_DATABASE_AS_FAILED_DELETE_DIRECTORY" : {
```

Comment:
We can probably consolidate this error message and the one below; i.e.:

```
"CANNOT_DROP_OBJECT_DUE_TO_FAILED_DIRECTORY_DELETION": {
  "message": [ "Unable to DROP %s %s as failed to delete its directory %s. " ]
}
```

Where this is called with:
```
new SparkException("CANNOT_DROP_OBJECT_DUE_TO_FAILED_DIRECTORY_DELETION",
      Array("DATABASE", dbDefinition.name, dbDefinition.locationUri.toString), e)
```